shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/application/DefaultApplicationClientConfig.java

```diff
@@ -59,4 +59,9 @@ public class DefaultApplicationClientConfig implements ApplicationClientConfig {
     )
     private String defaultLanguage;
 
+    @Schema(
+        description = "The configuration of the search field."
+    )
+    private DefaultSearchFieldConfig searchFieldConfig;
```

Comment:
I think this might become confusing as there is already the [DefaultApplicationToolConfig](https://github.com/terrestris/shogun/blob/main/shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/application/DefaultApplicationToolConfig.java) for tool configurations available. Is there any reason why we don't make use of it?